SampleOfferWithoutReward.sol

```diff
@@ -101,6 +136,14 @@ contract SampleOfferWithoutReward {
             paidOut += amount;
     }
 
+    function withdraw() {
```

Comment:
I don't understand why try to make a different function for this. Put your improvements, if any, inside `getDailyPayment()`